test/TestProjects/FirstTest-Typespec/FirstTest-Typespec.tsp

```diff
@@ -217,7 +217,7 @@ namespace Hello.Demo2 {
         @body action: RoundTripModel,
         @header contentType: "text/plain",
         @path p2: string
-    ): Thing;
+    ): RoundTripModel;
```

Comment:
Despite the name of this model says "roundtrip" but this model is not really a round trip model.
I changed these operations here to let the `RoundTripModel` to be roundtrip